### ISSUE TITLE
Validate Rust parser on query_log queries

### ISSUE THREAD

**Issue by nathanielc** [influxdata/flux#2283]
We want to run the Rust parser against all queries in the query_log that do not have an error. The intent is to check if the Rust parser errors on any existing queries and to investigate any potential issues.

DOD:

* We have run the Rust parser on non-erroring queries from the query_log.

**Comment by nathanielc** [comment]
NOTES:

Check out this https://github.com/influxdata/idpe/tree/master/cmd/qlfmt as a way to get started reading queries from the query log.

**Comment by dangilkerson** [comment]
Testing a week worth of production queries (3 million) a parsing error was found.

"Parsing Error: [Error { location: SourceLocation { file: Some(\"\"), start: Position { line: 3, column: 1 }, end: Position { line: 3, column: 8 }, source: Some(\"package\") }, message: \"invalid statement: package\" }, Error { location: SourceLocation { file: Some(\"\"), start: Position { line: 4, column: 1 }, end: Position { line: 4, column: 7 }, source: Some(\"import\") }, message: \"invalid statement: import\" }]"

Looks like it's due to the option line before imports

Examples:
option v = {}
import "strings"
import "influxdata/influxdb/v1"

// ----
option v = {timeRangeStart: -1h, timeRangeStop: now()}
import "strings"

**Comment by nathanielc** [comment]
Manually testing the Go and Rust parsers shows that both require the `imports` to come before the `options`. What is happening above is that the http request to the server provides the `options` as AST and the rest of the query as Flux source text. But `qlfmt` formats both the AST and the Flux source code together so it appears as if the `option v = ...` parts come before the imports, but in reality they are separate. We should update the `qlfmt` to place these in separate files since that is how they are represented in the http request.

TL;DR There is no issue here the Rust and Go parsers behave the same. The `options` error is just an artifact of the testing method.
